vue-frontend/public/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <meta name="author" content="canopas" />
-    <link rel="icon" href="<%= BASE_URL %>favicon-16x16.webp" />
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
```

Comment:
What is the standard size of favicon.ico?
Investigate it and replace our current ico according to that standard

Reply:
favicon.icon size is 16*16

Reply:
The whole image with background is 16*16, that's why it is showing too small.